dashboard2/dashboard2.md

```diff
@@ -14,15 +14,14 @@ This card displays the key properties of your project. They are the following:
 
 - **Project code**: An 8 character long identifier that's unique to your project. It can be used to track projects and is necessary when you ask help at our support email address.
 
-- **Source language**: The language you translate from. It can't be changed after the fact.
+- **Source language**: While it's possible to change it on the old Dashboard, we don't recommend doing so. The segmentation rules used depend on the source language. Changing them can cause weird issues with source segments. In most cases, it's better to keep it safe and leave the source language as is, even if it changes during the lifetime of the project.
```

Comment:
> Elsewhere, we note that it can be changed on the Old Dashboard.
>
> Why is it not recommended to do so, though?

This is what you said about its previous version. Seems to be contradicting. Would 

> The language you translate from. It can't be changed here but if you need to, you can email support and we'll be happy to change it for you after the necessary precautions.

be an OK middle ground?

Reply:
Good point, I can get behind that...